examples/default/main.tf

```diff
@@ -0,0 +1,62 @@
+provider "aws" {
+  region  = "us-east-1"
+  profile = "dev_ops"
```

Comment:
We should remove this profile since these are living examples that run as part of our unit testing so, we won't have this profile on our CI pipeline.
```suggestion
```

Reply:
Fixed!